src/drivers/wdm/queries/KeWaitLocal/KeWaitLocal.ql

```diff
@@ -1,15 +1,19 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 /**
+ * @id cpp/portedqueries/ke-wait-local
  * @name KeWaitLocal
+ * @description When the first argument to KeWaitForSingleObject is a local variable, the Mode parameter must be KernelMode.
+ * @security.severity: Low
+ * @impact: Exploitable Design
+ * @feature.area Multiple
+ * @repro.text The following code locations potentially contain KeWaitForSingleObject where the Mode parameter is not KernelMode.

```

Comment:
```suggestion
 * @repro.text The following code locations potentially contain a call to KeWaitForSingleObject waiting on a local kernel-mode object without setting the Mode parameter to KernelMode.

```

Reply:
How about: The following code locations contain a call to KeWaitForSingleObject while waiting for a local kernel-mode object, but the Mode parameter has not been set to KernelMode.

Reply:
Sounds good to me.